### ISSUE TITLE
Assetlib preview image does not load

### ISSUE THREAD

**Issue by LightShade12** [godotengine/godot#78402]
### Godot version

4.0.3.stable.official

### System information

Windows 10 - Godot.4.0.3.stable- Forward Renderer- Nvidia Geforce GTX 1650 4GB

### Issue description

Despite having working internet, godot 4 assetlib images do not load:
![image](https://github.com/godotengine/godot/assets/102753608/b61a76f7-2014-4a0b-a886-487831463fea)
assets list

![image](https://github.com/godotengine/godot/assets/102753608/36cf66b0-2831-4013-8b33-e64db2cec69c)
asset preview

I expect to see all the images as usual when using assetlib

### Steps to reproduce

I don't really know. No one else has reported this before. This is a fresh install of windows 10 and first install of Godot 4.
If it helps, I get broken image icon and am unable to see images on github repos on my browser as well.

### Minimal reproduction project

N/A

**Comment by AThousandShips** [comment]
> If it helps, I get broken image icon and am unable to see images on github repos on my browser as well.

Yes, this is likely the cause, either you are having connection problems or there's some issue with your computer, this is almost certainly not an error in Godot

**Comment by AThousandShips** [comment]
Do you still experience this problem?

**Comment by LightShade12** [comment]
> Do you still experience this problem?

i can't fix my web browser issue yet, so yeah..

**Comment by AThousandShips** [comment]
Are you behind a proxy or dual stack IPv4/IPv6 network?

**Comment by LightShade12** [comment]
> 

i dont any idea; in windows settings> newtork & internet> properties: i see addresses for both ipv4 and ipv6 and ipv6 dns
